Assets/MRTK/Examples/Experimental/Dwell/InstantDwellProfile.asset

```diff
@@ -13,7 +13,7 @@ MonoBehaviour:
   m_Name: InstantDwellProfile
   m_EditorClassIdentifier: 
   dwellTriggerPointerType: 4
-  dwellIntentDelay: 0
-  dwellStartDelay: 2
-  timeToCompleteDwell: 5
-  timeToAllowDwellResume: 1
+  dwellIntentDelay: 0.5
+  dwellStartDelay: 0.5
+  timeToCompleteDwell: 2
+  timeToAllowDwellResume: 2
```

Comment:
Are these changes intended? It looks like a lot of the profiles are now identical with the default settings

Reply:
I think @cre8ivepark made the changes (please see the commit history) and I tend to believe his judgements.

Reply:
@keveleigh Yes, I have updated them since they were too slow to be triggered.

Reply:
That works! @MaxWang-MS can we de-dupe the profiles that are now identical? Looks like at least InstantDwellProfile and ToggleDwellProfile.

Reply:
@keveleigh I would propose that we keep those two as it might cause confusion when the user tries to modify the behavior of a button only to find out another unrelated button also changes behavior. Currently the names of the profiles also clearly state what they do.